linera-rpc/src/grpc/client.rs

```diff
@@ -406,6 +406,9 @@ impl ValidatorNode for GrpcClient {
         let mut missing_hashes = hashes;
         let mut certs_collected = Vec::with_capacity(missing_hashes.len());
         loop {
+            if missing_hashes.is_empty() {
+                break;
+            }
```

Comment:
```suggestion
         while !missing_hashes.is_empty() {
```

Reply:
Done in [Add loop and early exit to download_certs_at_heights](https://github.com/linera-io/linera-protocol/pull/4373/commits/19e2145238a9094d1819a9fb3777d5ba00106573)